diff --git a/node_modules/vite/dist/node/chunks/config.js b/node_modules/vite/dist/node/chunks/config.js
index 2b39a20..ccdda2d 100644
--- a/node_modules/vite/dist/node/chunks/config.js
+++ b/node_modules/vite/dist/node/chunks/config.js
@@ -28793,6 +28793,7 @@ var EnvironmentPluginContainer = class {
 			const start = debugPluginTransform ? performance$1.now() : 0;
 			let result;
 			const handler = getHookHandler(plugin.transform);
+			if (!handler) continue;
 			try {
 				result = await this.handleHookPromise(handler.call(ctx, code, id, optionsWithSSR));
 			} catch (e$1) {
